[GCI-riskycase] AC-639: Set Code Style (#651)
openmrs/openmrs-contrib-android-client: README.md

### Code Style

> The coding conventions used by OpenMRS are outlined [here](https://wiki.openmrs.org/display/docs/Coding+Conventions).

Reviewer: Reword this to point to https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+IntelliJ#DeveloperHow-ToSetupAndUseIntelliJ-SetupCodeStyleAndFormatForOpenMRS

> Please follow the below instructions while contributing to the development of the app

Reviewer: ```suggestion
Please follow the below instructions while contributing:
```

1. When project is opened with Android Studio, go to File > Settings > Editor > Code Style

> 2. Select Project from the Scheme dropdown

Reviewer: You should be bolding options that user can pick, **like so**

> 3. Make sure that hard wrap is the same under Wrapping and Bases in Java and Other in XML language settings (currently 125)

Reviewer: This statement is unclear. Specify the hard wrap specifically.
```suggestion
3. Make sure that **hard wrap** is set to **125** for Java and XML language settings.
```
